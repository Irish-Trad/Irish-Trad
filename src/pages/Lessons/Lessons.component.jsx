import React from 'react'

import { MainContainer, H1, H2, Text, Container, Link } from './Lessons.styles'

const Lessons = () => (
  <MainContainer>
    <H1>Lessons</H1>
    <H2>Preparation:</H2>
    <Container>
      <Link href="../pages/lessons/stringing-the-instrument.html">
        How to string your instrument
      </Link>
      <Text>
        Knowing how to string your instrument is an essential skill every
        musician needs to learn. It can be a daunting task for beginners because
        of uncertainty and fear of damaging the instrument. This section covers
        how to string the instrument and the different options available when
        buying strings.
      </Text>
      <Link href="../pages/lessons/tuning-the-instrument.html">
        How to tune your instrument
      </Link>
      <Text>
        When playing with other musicians it is vital that your instrument is in
        tune. This section covers how to use a chromatic tuner and the different
        tuning configurations available to the Irish Bouzouki.
      </Text>
      <Link href="../pages/lessons/how-to-read-tablature.html">
        How to read tablature
      </Link>
      <Text>
        Tablature is a form of music notation structured towards fretted
        instruments. It is regarded as one of the quickest methods for teaching
        beginners how to start making music. This section covers how to read
        tablature.
      </Text>
    </Container>
    <H2>Useful Websites:</H2>
    <Container>
      <Link
        href="https://thesession.org/"
        target="_blank"
        rel="noopener noreferrer"
      >
        The Session
      </Link>
      <Text>The best resource for Irish traditional sheet music.</Text>
      <Link
        href="http://irish-music.net/"
        target="_blank"
        rel="noopener noreferrer"
      >
        Irish-Music
      </Link>
      <Text>
        Useful website that provides links to many resources for traditional
        Irish instruments.
      </Text>
      <Link
        href="http://mandolessons.com/"
        target="_blank"
        rel="noopener noreferrer"
      >
        Mando Lessons
      </Link>
      <Text>
        Mandolin focused website providing high quality video lessons.
      </Text>
    </Container>
  </MainContainer>
)

export default Lessons
