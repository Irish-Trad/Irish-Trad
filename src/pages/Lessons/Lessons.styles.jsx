import styled from 'styled-components'

export const MainContainer = styled.div``

export const H1 = styled.h1``

export const H2 = styled.h2``

export const Text = styled.p``

export const Container = styled.div``

export const Link = styled.a``
