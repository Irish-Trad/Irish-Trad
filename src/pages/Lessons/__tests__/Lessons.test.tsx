import { render, cleanup } from '@testing-library/react'
import Lessons from '../Lessons.component'
import { HashRouter } from 'react-router-dom'

const renderComponent = () =>
  render(
    <HashRouter>
      <Lessons />
    </HashRouter>
  )

describe('Lessons', () => {
  afterEach(() => {
    cleanup()
  })

  it('Snapshot', () => {
    const snapshot = renderComponent()

    expect(snapshot).toMatchSnapshot()
  })

  it('Title of the page is Lessons', () => {
    const { getByText } = renderComponent()

    expect(getByText('Lessons')).toBeInTheDocument()
  })
})
