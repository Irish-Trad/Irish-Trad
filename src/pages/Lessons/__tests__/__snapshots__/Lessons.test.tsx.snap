// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lessons Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="sc-bdfBwQ fyfzYh"
      >
        Lessons
      </h1>
      <section
        class="sc-pFZIQ kpHvOQ"
      >
        <h2
          class="sc-iqHYGH fPstTD"
        >
          Preparation:
        </h2>
        <a
          class="sc-crrsfI iXdlXA"
          href="#/lessons/how-to-string"
        >
          <p
            class="sc-dQppl"
          >
            How to string your instrument
          </p>
        </a>
        <p
          class="sc-dlfnbm gBOapU"
        >
          Knowing how to string your instrument is an essential skill every musician needs to learn. It can be a daunting task for beginners because of uncertainty and fear of damaging the instrument. This section covers how to string the instrument and the different options available when buying strings.
        </p>
        <a
          class="sc-crrsfI iXdlXA"
          href="#/lessons/how-to-tune"
        >
          <p
            class="sc-dQppl"
          >
            How to tune your instrument
          </p>
        </a>
        <p
          class="sc-dlfnbm gBOapU"
        >
          When playing with other musicians it is vital that your instrument is in tune. This section covers how to use a chromatic tuner and the different tuning configurations available to the Irish Bouzouki.
        </p>
        <a
          class="sc-crrsfI iXdlXA"
          href="#/lessons/how-to-read-tab"
        >
          <p
            class="sc-dQppl"
          >
            How to read tablature
          </p>
        </a>
        <p
          class="sc-dlfnbm gBOapU"
        >
          Tablature is a form of music notation structured towards fretted instruments. It is regarded as one of the quickest methods for teaching beginners how to start making music. This section covers how to read tablature.
        </p>
        <h2
          class="sc-iqHYGH fPstTD"
        >
          Useful Websites:
        </h2>
        <a
          class="sc-bqyKva itefUV"
          href="https://thesession.org/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <p
            class="sc-dQppl"
          >
            The Session
          </p>
        </a>
        <p
          class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
        >
          The best resource for traditional Irish sheet music.
        </p>
        <a
          class="sc-bqyKva itefUV"
          href="https://www.atuneaweek.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <p
            class="sc-dQppl"
          >
            A Tune A Week
          </p>
        </a>
        <p
          class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
        >
          A great resource to keep you motivated and help choose what tune to learn next.
        </p>
        <a
          class="sc-bqyKva itefUV"
          href="https://irish-music.net/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <p
            class="sc-dQppl"
          >
            Irish-Music
          </p>
        </a>
        <p
          class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
        >
          Useful website that provides links to many resources for traditional Irish instruments.
        </p>
        <a
          class="sc-bqyKva itefUV"
          href="https://www.mandolessons.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <p
            class="sc-dQppl"
          >
            Mando Lessons
          </p>
        </a>
        <p
          class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
        >
          Mandolin focused website providing high quality video lessons.
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <h1
      class="sc-bdfBwQ fyfzYh"
    >
      Lessons
    </h1>
    <section
      class="sc-pFZIQ kpHvOQ"
    >
      <h2
        class="sc-iqHYGH fPstTD"
      >
        Preparation:
      </h2>
      <a
        class="sc-crrsfI iXdlXA"
        href="#/lessons/how-to-string"
      >
        <p
          class="sc-dQppl"
        >
          How to string your instrument
        </p>
      </a>
      <p
        class="sc-dlfnbm gBOapU"
      >
        Knowing how to string your instrument is an essential skill every musician needs to learn. It can be a daunting task for beginners because of uncertainty and fear of damaging the instrument. This section covers how to string the instrument and the different options available when buying strings.
      </p>
      <a
        class="sc-crrsfI iXdlXA"
        href="#/lessons/how-to-tune"
      >
        <p
          class="sc-dQppl"
        >
          How to tune your instrument
        </p>
      </a>
      <p
        class="sc-dlfnbm gBOapU"
      >
        When playing with other musicians it is vital that your instrument is in tune. This section covers how to use a chromatic tuner and the different tuning configurations available to the Irish Bouzouki.
      </p>
      <a
        class="sc-crrsfI iXdlXA"
        href="#/lessons/how-to-read-tab"
      >
        <p
          class="sc-dQppl"
        >
          How to read tablature
        </p>
      </a>
      <p
        class="sc-dlfnbm gBOapU"
      >
        Tablature is a form of music notation structured towards fretted instruments. It is regarded as one of the quickest methods for teaching beginners how to start making music. This section covers how to read tablature.
      </p>
      <h2
        class="sc-iqHYGH fPstTD"
      >
        Useful Websites:
      </h2>
      <a
        class="sc-bqyKva itefUV"
        href="https://thesession.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="sc-dQppl"
        >
          The Session
        </p>
      </a>
      <p
        class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
      >
        The best resource for traditional Irish sheet music.
      </p>
      <a
        class="sc-bqyKva itefUV"
        href="https://www.atuneaweek.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="sc-dQppl"
        >
          A Tune A Week
        </p>
      </a>
      <p
        class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
      >
        A great resource to keep you motivated and help choose what tune to learn next.
      </p>
      <a
        class="sc-bqyKva itefUV"
        href="https://irish-music.net/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="sc-dQppl"
        >
          Irish-Music
        </p>
      </a>
      <p
        class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
      >
        Useful website that provides links to many resources for traditional Irish instruments.
      </p>
      <a
        class="sc-bqyKva itefUV"
        href="https://www.mandolessons.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <p
          class="sc-dQppl"
        >
          Mando Lessons
        </p>
      </a>
      <p
        class="sc-dlfnbm sc-jrAGrp gBOapU kJTqhj"
      >
        Mandolin focused website providing high quality video lessons.
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
